lib/build/geo_feature_builder.dart

```diff
@@ -0,0 +1,158 @@
+        listRings.add(refer('Ring').newInstance([literalList([points])]));
+    }
+
+    if(listRings.isEmpty){
+      inner = literalList([[]]);
+    }
+    else{
+      inner = literalList([listRings]);
+    }
+    return refer('Polygon').newInstance([refer(outer.toString()), refer(inner.toString())]);
+}
+
+  String createClass(){
+    final library = Library((b) => b
+      ..directives.add(Directive.import('package:offline_geocoder/offline_geocoder.dart'))
+      ..directives.add(Directive.import('package:latlong2/latlong.dart'))
+      ..body.addAll([geoFeatureClass().build(), geoCoderClass().build()]));
+
+    final dartfmt = DartFormatter();
+    return dartfmt.format('${library.accept(DartEmitter.scoped())}').toString();
+  }
+
+  ClassBuilder geoFeatureClass(){
+
+    final classFields = propertiesToExtract.entries.map((entry) { 
+      return Field((b) => b
+        ..modifier = FieldModifier.final$
+        ..name = ReCase(entry.key).camelCase 
+        ..type = Reference(entry.value,'dart:core').type);
+    });
+
+    final constructorParameters = propertiesToExtract.entries.map((entry) {
+      return Parameter((b) => b
+        ..name = ReCase(entry.key).camelCase
+        ..named = true
+        ..toThis = true);
+    });
+
+    return ClassBuilder()
+      ..name = 'GeoFeature'
+      ..extend = refer('GeoFeatureBase')
+      ..fields.addAll(classFields)
+      ..constructors.add(Constructor((b) => b
+        ..constant = true
+        ..requiredParameters.addAll(constructorParameters)
+        ..requiredParameters.add(Parameter((b) => b
+          ..name = 'area'
+          ..named = true
+          ..toSuper = true))
+      ));
+  }
+
+  ClassBuilder geoCoderClass(){
+
+     final geoFeatures = finalGeoFeatureList.map((e) {
+      final List<Reference> geoFeatureParameters = [];
+      for (final key in propertiesToExtract.keys) {
+          propertiesToExtract[key] == 'String' 
+          ? geoFeatureParameters.add(refer("'${e[ReCase(key).camelCase]}'"))
+          : geoFeatureParameters.add(refer('${e[ReCase(key).camelCase]}'));
+      }
+      geoFeatureParameters.add(e['area']);
+      return refer('GeoFeature').newInstance(geoFeatureParameters);
+    }).toList();
+
+    print(geoFeatures.toString());
+    return ClassBuilder()
+      ..name = 'GeoCoder'
+      ..constructors.add(Constructor((b) => b))
+      ..fields.add(Field((b) => b
+        ..name = 'Test'
+        ..modifier = FieldModifier.final$
+        ..assignment = literalList([geoFeatures], refer('List<GeoFeature>')).code))
+      ..methods.add(Method((b) => b
+        ..name = 'getFromLocation'
+        //..static = true 
+        ..returns = refer('GeoFeature?')
+        ..requiredParameters.add(Parameter((b) => b
+          ..name = 'point'
+          ..type = refer('LatLng')))
+        ..body = refer('firstGeoFeatureContainingPoint<GeoFeature>').call([refer('point'), refer('geoFeatures')]).code))
+    ;
+  }
```

Comment:
return `Class` here instead of `ClassBuilder` by calling `.build()` at the end inside each function